.storybook/test-storybook-canvas.config.js

```diff
@@ -61,6 +67,20 @@ const configuration = {
             }),
             {},
         ),
+        "Core components/GeoPushpinChart/Config/Map": mapStories.reduce(
```

Comment:
should it be 
`"Core components/GeoPushpinChart/Config/Viewport": viewportStories.reduce(`? 
We don’t have `map` property in `IGeoConfig`

Reply:
Updated.
I followed by Map under Configuration Panel.